examples/video-chat/screens/PreviewScreen/index.tsx

```diff
@@ -65,27 +69,41 @@ const PreviewScreen = ({ navigation, route }: Props) => {
     setIsMicrophoneOn(!isMicrophoneOn);
   };
 
-  const toggleCamera = () => {
-    setIsCameraOn(!isCameraOn);
-  };
-
   const toggleSwitchCamera = () => {
-    setCurrentCamera(
+    const camera =
       availableCameras.current.find(
         (device) => device.isFrontFacing !== currentCamera?.isFrontFacing,
-      ) || null,
-    );
+      ) || null;
+    if (camera) {
+      switchCamera(camera.id);
+      setCurrentCamera(camera);
+    }
   };
 
   useEffect(() => {
     getCaptureDevices().then((devices) => {
       availableCameras.current = devices;
+
+      startCamera({
+        simulcastConfig: {
+          enabled: true,
+          activeEncodings:
+            Platform.OS === 'android' ? ['l', 'm', 'h'] : ['l', 'h'],
```

Comment:
Does anyone knows why there is different set for ios and android?

Reply:
yes, I know why
will add a comment in the code or the docs
on iOS on some devices if using H264 hardware encoder there is a hardware limit of 3 encoders (https://github.com/twilio/twilio-video-ios/issues/17) 
every simulcast layer is an encoder
if you start screencast that's one more encoder
3 simulcast layers + 1 screencast layer = 4 encoders
4 > 3, so screencast is going to silently break

afaik android has no such limitations

Reply:
In that case, I think we should make this change on SDK level 🤔